.circleci/config.yml

```diff
@@ -167,10 +165,10 @@ jobs:
       - install-py-dev-env
       - run: cp ~/project/.circleci/circleci_settings.py creme/project_settings.py
       - run: cp ~/project/.circleci/sqlite3_settings.py creme/local_settings.py
-      - run: python manage.py migrate
-      - run: python manage.py creme_populate
-      - run: python manage.py generatemedia
-      - run: COVERAGE_PROCESS_START=setup.cfg coverage run --source creme/ manage.py test --noinput --parallel=8
+      - run: python creme/manage.py migrate
```

Comment:
`- run: MAKE_NPROCS=8 make test-cov` ?

Reply:
By the way I am not sure that the `coverage html` is really useful here... `coverage report` seems more useful.

Reply:
coverage html generates html reports that are then saved as build artifacts, and can be browse.
Ex: https://1442-264866391-gh.circle-artifacts.com/0/artifacts/coverage_html/index.html

Reply:
Oh ok, I though it is was lost in the space. Can you print the link ?

Reply:
It already printed, look :D ! 
![image](https://user-images.githubusercontent.com/1230876/87136616-0acb3e00-c29c-11ea-9e41-4d26848160ef.png)